src/CodeGen/ODataT4CodeGenerator.ttinclude

```diff
@@ -3420,6 +3440,18 @@ namespace <#= fullNamespace #>
 
     internal override void WriteGeneratedEdmModel(string escapedEdmxString)
     {
+
+        string path = this.context.TempFilePath; 
+        
+        if(!String.IsNullOrEmpty(path))

```

Comment:
We usually have space between if and (. Same comment for the other cases in this file